doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
What would be the point? There would be no reason to use `ROOT_SLASH` instead of `ROOT_DIR`, and then we would have three macros for one thing.